llvm/lib/Object/OffloadBundle.cpp

```diff
@@ -260,11 +299,233 @@ static std::string formatWithCommas(unsigned long long Value) {
 }
 
 llvm::Expected<std::unique_ptr<llvm::MemoryBuffer>>
-CompressedOffloadBundle::decompress(llvm::MemoryBufferRef &Input,
+CompressedOffloadBundle::compress(llvm::compression::Params P,
+                                  const llvm::MemoryBuffer &Input,
+                                  uint16_t Version, bool Verbose) {
+  if (!llvm::compression::zstd::isAvailable() &&
+      !llvm::compression::zlib::isAvailable())
+    return createStringError(llvm::inconvertibleErrorCode(),
```

Comment:
```suggestion
    return createStringError(
```
Pretty sure there's a constructor that does the error code for you.

Reply:
This is copied code from the clang-offload-bundler API, but I agree and can make this change.

Reply:
For the fixes from copy/paste, can we create a separate PR to go back and fix it in the original APIs?